build.gradle

```diff
@@ -139,7 +143,7 @@ apply from: file('gradle/maven/defaults-maven.gradle')
 
 // IDE support, settings and specials.
 apply from: file('gradle/ide/intellij-idea.gradle')
-apply from: file('gradle/ide/eclipse.gradle')
+// apply from: file('gradle/ide/eclipse.gradle')
```

Comment:
do we remove it then?

Reply:
Good catch, this slipt into the PR without me noticing it before I create the PR. I have to see if I can build the project with the `eclipse.gradle` file loaded, it caused some issues on my environment.

Reply:
So I uncommented it fixed some version references and a null error that was thrown on my side. Hopefully this works the same now for eclipse users.